pkg/db/topology.go

```diff
@@ -315,11 +316,11 @@ func DeleteComponentsOfTopology(topologyID string, deleteTime time.Time) error {
 			logger.Errorf("Error deleting component[%s] relationship for component %v", component.ID, err)
 		}
 
-		if component.ComponentChecks != nil {
-			if err := DeleteInlineCanariesForComponent(component.ID.String(), deleteTime); err != nil {
-				logger.Errorf("Error deleting inline canaries for component %s: %v", component.ID, err)
-			}
-		}
+		// if component.ComponentChecks != nil {
```

Comment:
duty's `Component` model doesn't have a `ComponentChecks` field. 
This field was removed some time ago because it was determined that it was not being utilized anywhere.